payment-failure/src/main/scala/com/gu/identity/paymentfailure/SqsService.scala

```diff
@@ -0,0 +1,37 @@
+package com.gu.identity.paymentfailure
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentialsProvider}
+import com.amazonaws.services.lambda.runtime.events.SQSEvent
+import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder
+import com.amazonaws.services.sqs.model.{DeleteMessageRequest, DeleteMessageResult}
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.Decoder
+import io.circe.parser.decode
+import io.circe.generic.semiauto.deriveDecoder
+import cats.syntax.either._
+
+class SqsService extends StrictLogging {
+
+  def parseSingleMessage(sqsMessage: SQSMessage): Either[Throwable, IdentityBrazeEmailData] = {
+    logger.info(s"attempting to parse message body ${sqsMessage.getBody}")
+    implicit val identityBrazeEmailDataDecoder: Decoder[IdentityBrazeEmailData] = deriveDecoder[IdentityBrazeEmailData]
+    decode[IdentityBrazeEmailData](sqsMessage.getBody)
+  }
+
+  def deleteMessage(message: SQSEvent.SQSMessage, config: Config): Either[Throwable, DeleteMessageResult] = {
```

Comment:
Config should be passed as a class constructor (same reasons as previous comment).